testing/unittests/unittests.cpp

```diff
@@ -1052,5 +1052,47 @@ class RelaxedAtomicTimingTest : public CppUnit::TestFixture
 
 CPPUNIT_TEST_SUITE_REGISTRATION( RelaxedAtomicTimingTest );
 CPPUNIT_TEST_SUITE_NAMED_REGISTRATION( RelaxedAtomicTimingTest, "RelaxedAtomicTimingTest" );
+#include "jlzw.hpp"
+class compressToBufferTest : public CppUnit::TestFixture
+{
+    CPPUNIT_TEST_SUITE( compressToBufferTest  );
+        CPPUNIT_TEST(testRun);
+    CPPUNIT_TEST_SUITE_END();
+
+    void testRun()
+    {
+        MemoryBuffer x;
+        compressToBuffer(x, 251, 
```

Comment:
So it's clear length is correct (if ever changed etc), would be clearer if string was declared separately, and strlen instead of constant 251 was used here.